Sources/Views/MessageLabel.swift

```diff
@@ -114,7 +114,9 @@ open class MessageLabel: UILabel {
     }
 
     open override var intrinsicContentSize: CGSize {
-        var size = super.intrinsicContentSize
+        let maxWidth = preferredMaxLayoutWidth - textInsets.horizontal
+        let constraintBox = CGSize(width: maxWidth, height: .greatestFiniteMagnitude)
+        var size = textStorage.boundingRect(with: constraintBox, options: [.usesLineFragmentOrigin, .usesFontLeading], context: nil).integral.size
```

Comment:
@marius-serban You're finding these issues because you're using autolayout/xibs with MessageKit? 

I can see how this is necessary to solve this issue 👍 This is a really heavy operation though, so I'd like to see what impact this has on performance.

Reply:
Indeed, I expect `intrinsicContentSize` was calling it too but maybe only in a multi-line width constrained scenario and probably caching the result. Some optimisations could definitely be done here, this is a bare-bones not-that-efficient but correct implementation.

The reason I run into this issues is because I'm using a custom cell, non-xib but laid out with autolayout. The content is dynamic, each label could go multi-line and I didn't want to worry about computing the size manually at this point, feels like an early optimisation for my use case.

<img width="302" alt="screen shot 2018-09-27 at 14 00 55" src="https://user-images.githubusercontent.com/1109961/46937445-3bdff580-d059-11e8-99a1-384a8f890fc2.png">

Reply:
Still haven't had time to investigate this but I wanted to tag #824. These two methods should be consistent if it gets merged